pyQuARC/code/downloader.py

```diff
@@ -55,9 +50,9 @@ def _construct_url(self):
             (str) The URL constructed based on the concept ID
         """
 
-        concept_id_type = Downloader._concept_id_type(self.concept_id)
         constructed_url = Downloader.BASE_URL.format(
-            concept_id=self.concept_id, metadata_format=self.metadata_format
+            concept_id=self.concept_id,
+            metadata_format=self.metadata_format if not self.metadata_format.startswith("umm-") else "umm-json"
```

Comment:
Do we even need this check?

Reply:
yeah
initially we only used `umm-json` for umm-json files (since we only supported collections), but now we use `umm-g` for umm-json granules and `umm-c` for the collections. so if it's `umm-<something>` we change it to umm-json

Reply:
```
medata_format = self.metadata_format
if ...
    metadata_format = "umm-json"
```